### PR TITLE
update build procedure in doc/tutorial_en(zh).md

### PR DESCRIPTION
**Pull request by lrita**
update build procedure in doc/tutorial_en(zh).md
with #143

### CODE REVIEW

File: doc/tutorial_en.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Install Go please check [this document](https://github.com/astaxie/build-web-app
 go get github.com/reborndb/reborn
 cd $GOPATH/src/github.com/reborndb/reborn
 ./bootstrap.sh
+make build
```
Comment (siddontang) [inline doc/tutorial_en.md]: you can use make directly.

**Comment by lrita** [comment]
因为deploy.tar下文并没描述，所以觉得make build更符合上下文。

**Comment by siddontang** [comment]
@qiuyesuifeng what's your opinion?

**Comment by qiuyesuifeng** [comment]
@lrita  Thanks.
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**